components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   resourceSelector:
     matchLabels:
       monitoring.rhobs/stack: appstudio-federate-ms
-  logLevel: info # use debug for verbose logs
+  logLevel: debug # use debug for verbose logs
```

Comment:
Is this a *permanent* desired logging level for *both* `staging` and  `production` environments ?

Reply:
Nice catch, it was for our debugging